R/nitrogen_efficiency.R

```diff
@@ -145,22 +154,33 @@ calc_n_efficiency <- function(B_LU_BRP, B_SOILTYPE_AGR, B_GWL_CLASS, B_AER_CBS,
 #' This function gives an indicator value for nitrogen use efficiency calculated by \code{\link{calc_n_efficiency}}, this function makes use of \code{\link{ind_nretention}}
 #' 
 #' @param D_NLEACH (numeric) The value of N leaching calculated by \code{\link{calc_n_efficiency}}
+#' @param leaching_to (character) whether it evaluates N leaching to groundwater ("gw") or to surface water ("ow")
 #' 
 #' @examples 
-#' ind_n_efficiency(D_NLEACH = 50)
-#' ind_n_efficiency(D_NLEACH = c(5,15,25,75))
+#' ind_n_efficiency(D_NLEACH = 50, leaching_to = 'gw')
+#' ind_n_efficiency(D_NLEACH = c(5,15,25,75), leaching_to = 'ow')
 #'  
 #' @return 
 #' The evaluated score for the soil function to enhance the nitrogen use efficiency. A numeric value between 0 and 1.
 #'         
 #' @export
-ind_n_efficiency <- function(D_NLEACH){
-  
-    # Evaluate the N retention for groundwater 
-    I_W_NGW <- ind_nretention(D_NLEACH,'gw')
+ind_n_efficiency <- function(D_NLEACH,leaching_to){
```

Comment:
To prevent a breaking cahnge set the default of `leaching_to` to `gw`, as that value was previously used